strategy_node:
  ros__parameters:
    waypoints:
      - [0.54, -0.26,   1.57, # Вторая куча банок
         0.30, -0.82, -1.57, # Место второй сборки 
         0.92, -0.42, -1.60, # Первая куча банок
         0.70, -1.01, -1.57, # Место первой сборки
         0.9, 0.2, 1.57, # Место ожидания
         1.2,  0.97, 1.57 # Место финиша
        ]  # x, y, theta ==> Каждые 3 числа - одна точка.
      # - [-1.22, 0.89, 0.0] # На место финиша
    elevator_order:
     - [3, 1, 2, 1, 2, 0] # Порядок команд элеватора
    #  - [3, 1, 2, 0] # Порядок команд элеватора
    time_until_end: 93
    obstacle_release: 
      - ["center_material_left", "upper_material_left"]
    term_order: 
      - [20,12,12,10]
      # - [20, 8, 10]